### DISCUSSION TITLE
Mullvad as VPN provider

### DISCUSSION THREAD

**Discussion by muammar** [RaspAP/raspap-docker#28] [General]
How can I change https://github.com/RaspAP/raspap-docker/blob/master/Dockerfile#L29 to install Mullvad as a VPN provider? I know the option appears when I install using `bash`, but I have not figured out how to pass a flag explicitly to install this option on Docker.

**Comment by billz** [comment]
At the moment there's no [installer switch](https://docs.raspap.com/quick/#switches) to specify a VPN provider.
This could however be added in a future release, or you're welcome to open a PR in the main repo.

**Comment by billz** [comment]
The VPN provider non-interactive option is now available in the installer:
https://docs.raspap.com/quick/#-e-provider-value

**Reply by muammar** [reply]
That's amazing. Thank you. I did not have the bandwidth to do it. I'll try it out and let you know. But now I remember that Mullvad was not working for me when using Debian Bookworm and the latest version of Raspap. It works fine when the Debian version is bullseye. I'll open a report later. Thanks for your help!

**Reply by billz** [reply]
That's fine, but please be aware that there's no support whatsoever for provider CLI's. Thanks
